src/content-handlers/iiif/modules/uv-moreinforightpanel-module/MoreInfoRightPanel.ts

```diff
@@ -35,6 +35,7 @@ export class MoreInfoRightPanel extends RightPanel<MoreInfoRightPanelConfig> {
 
     this.$metadata = $('<div class="iiif-metadata-component"></div>');
     this.$main.append(this.$metadata);
+    this.$main.attr('tabindex', 0);
```

Comment:
Might it also make sense to add: `this.$main.attr('aria-label', this.config.content.title)` so the focusable element has an accessible title? (That's genuinely a question; I'm not totally sure -- have you tested how this behaves in a screen reader?).

Reply:
Good point. Will add it, although the divs will either need a role or changing to something like <article> in order to be readable by Jaws/Nvda I think.